src/common/containers/landing.js

```diff
@@ -65,67 +67,33 @@ class Landing extends Component {
         </section>
 
         <section className={ styles.section }>
-          <HeadingSix>
+          <HeadingSix align='centered'>
```

Comment:
This should be `center`.

This heading is centered even with this typo because HeadingSix is always centered according to its styles (`.h6` class).

`.h6` seems to have very specific (non vanilla) styles ATM, so maybe it all should be some variant of Heading component.